common/src/main/java/us/myles/ViaVersion/util/ReflectionUtil.java

```diff
@@ -64,6 +64,13 @@ public static void set(Object o, String f, Object value) throws NoSuchFieldExcep
         field.set(o, value);
     }
 
+    public static Class<?> getClass(String className) throws ClassNotFoundException {
+        try {
+            return Class.forName(className);
+        } catch (ClassNotFoundException ignored) {
+        }
+        return Class.forName("org.bukkit.craftbukkit.libs."+className);
```

Comment:
ReflectionUtil seems to be for mainly java reflection, this code is somewhat bound to CraftBukkit